### PR TITLE
Upgrade to EasyCodingStandard 2.2

### PR DESCRIPTION
**Pull request by TomasVotruba**
- upgrade to ECS 2.2
- fix travis checking code (was not running at all)
- check `/src` directory as well
- fix cs (of course :))

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -3,9 +3,8 @@
 	"license": "MIT",
 	"require": {
 		"php": "^7.1",
-		"symplify/easy-coding-standard": "^2.1.7",
-		"friendsofphp/php-cs-fixer": "^2.4",
-		"slevomat/coding-standard": "^3.0"
+		"symplify/easy-coding-standard": "^2.2.1",
+		"friendsofphp/php-cs-fixer": "^2.4"
```
Comment (TomasVotruba) [inline composer.json]: `slevomat/coding-standard` is required by ecs by default, it uses few of it's utilies and defacto must-have standard

Comment (dg) [inline composer.json]: Oh, you can give it away.

Comment (TomasVotruba) [inline composer.json]: Ok. I think `friendsofphp/php-cs-fixer` can be removed too, as it is the core part of `ecs`

**Comment by TomasVotruba** [comment]
Ready to merge!

**Comment by dg** [comment]
Great, thanks!

**Pull request merged**